mne/chpi.py

```diff
@@ -1003,7 +1003,7 @@ def _calculate_chpi_coil_locs(raw, t_step_min=0.1, t_step_max=10.,
         #
         # 1. Fit amplitudes for each channel from each of the N cHPI sinusoids
         #
-        sin_fit = _fit_cHPI_amplitudes(raw, time_sl, hpi, fit_time)
+        sin_fit = _fit_chpi_amplitudes(raw, time_sl, hpi, fit_time)
```

Comment:
for example, everywhere else in this file we abide by the `chpi_` naming, so this one was essentially mis-named